src/Tizen.Account.AccountManager/Tizen.Account.AccountManager/SafeAccountHandle.cs

```diff
@@ -60,7 +60,6 @@ public override bool IsInvalid
         /// <returns>true if the handle is released successfully</returns>
         protected override bool ReleaseHandle()
         {
-            Interop.Account.Destroy(this.handle);
             this.SetHandle(IntPtr.Zero);
```

Comment:
If it does not has owner ship of handle 
please use `public SafeAccountHandle(IntPtr existingHandle, bool ownsHandle) ` with ownsHandle = false

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.safehandle.-ctor?view=netframework-4.7.2

>If the ownsHandle parameter is false, ReleaseHandle is never called; thus, it is not recommended to use this parameter value as your code may leak resources.

Reply:
Yes false is used with this API.

Reply:
If you remove Destroy method call in ReleaseHandle, 

It will cause memory leak when use `CreateAccount`
``` c#
        public static Account CreateAccount()
        {
            SafeAccountHandle handle;
            AccountError err = (AccountError)Interop.Account.Create(out handle);
            if (err != AccountError.None)
            {
                throw AccountErrorFactory.CreateException(err, "Failed to create new error.");
            }

            return new Account(handle);
        }
```

Reply:
Why do you think the reason for the crash is Interop.Account.Destroy?

Reply:
As per the test reports of native test suite, we found that crash occurs immediately after account_destroy() call. JIRA issues - TFDF-272 & TFDF-270 are raised for the same.

Reply:
Do you know reproduce step?
When a crash occurred?

Reply:
Please check this solution (https://github.com/myroot/TizenFX/commit/8d60061e591196a830f7dace1c3516b0c294ff57) and please test it

``` c#
internal static extern int QueryAccountById(int accountId, ref SafeAccountHandle handle);
```
It will free passed handle. but ownership does not passed by C# layer. so it cause double free